deepchem/models/torch_models/tests/test_molgan_layers.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import pytest
+
+try:
+    import torch
+    # import torch.nn as nn
+    import torch.nn.functional as F
+    # , MolGANMultiConvolutionLayer, MolGANAggregationLayer, MolGANEncoderLayer
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+
+
+class test_molgan_layers(unittest.TestCase):
```

Comment:
classes should be in camel case (TestMolganLayers)

Reply:
done (class was removed as it is a pytest now)